quic/server/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ add_library(
   handshake/DefaultAppTokenValidator.cpp
   handshake/StatelessResetGenerator.cpp
   state/ServerStateMachine.cpp
+
+  # Fizz specific parts, will be split in its own lib eventually.
+  ../fizz/server/handshake/FizzServerQuicHandshakeContext.cpp
```

Comment:
While this works, we usually follow the directory structure. Could you add a CMakeLists.txt under quic/fizz/server and link that target here? Here's how it's done for client (https://github.com/facebookincubator/mvfst/blob/master/quic/fizz/client/CMakeLists.txt).

Reply:
This is not possible yet as the two pieces of code are too tightly coupled, but this is what I'm aiming for.